setup.sh

```diff
@@ -200,48 +201,39 @@ function reset() {
 }
 
 function config() {
-
-    echo "-------------------------"
-    echo "Please use 'freqtrade new-config -c config.json' to generate a new configuration file."
-    echo "-------------------------"
+    echo_block "Please use 'freqtrade new-config -c config.json' to generate a new configuration file."
 }
 
 function install() {
-    echo "-------------------------"
-    echo "Installing mandatory dependencies"
-    echo "-------------------------"
+    
+    echo_block "Installing mandatory dependencies"
 
-    if [ "$(uname -s)" == "Darwin" ]
-    then
+    if [ "$(uname -s)" == "Darwin" ]; then
         echo "macOS detected. Setup for this system in-progress"
         install_macos
-    elif [ -x "$(command -v apt-get)" ]
-    then
+    elif [ -x "$(command -v apt-get)" ]; then
         echo "Debian/Ubuntu detected. Setup for this system in-progress"
         install_debian
+    elif [ -x "$(command -v yum)" ]; then
+        echo "Red Hat/CentOS detected. Setup for this system in-progress"
+        install_redhat
     else
         echo "This script does not support your OS."
-        echo "If you have Python3.6 or Python3.7, pip, virtualenv, ta-lib you can continue."
+        echo "If you have Python version 3.6 - 3.9, pip, virtualenv, ta-lib you can continue."
```

Comment:
as you've touched this ... freqtrade doesn't support 3.6 anymore - so i'd change this to 3.7...